api/worker.go

```diff
@@ -28,6 +28,11 @@ type AccountsUnlockHandlerRequest struct {
 	LockID uint64 `json:"lockID"`
 }
 
+type PackedObjectInfo struct {
```

Comment:
our `ObjectMetadata` type looks as follows:
```go
type ObjectMetadata struct {
	Name string `json:"name"`
	Size int64  `json:"size"`
}
```

I think `path` makes sense but should we change `length` to `size` maybe? Also probably either make both `uint64` or `int64` so the API overall is more consistent

Reply:
Yeah I'll change that once we are clear on the endpoint.